src/main/java/com/metamx/emitter/core/Emitters.java

```diff
@@ -72,6 +72,7 @@ else if (props.getProperty(HTTP_EMITTER_PROP) != null) {
     httpMap.put("recipientBaseUrl", baseUrl);
     httpMap.put("flushMillis", Long.parseLong(props.getProperty("com.metamx.emitter.flushMillis", "60000")));
     httpMap.put("flushCount", Integer.parseInt(props.getProperty("com.metamx.emitter.flushCount", "300")));
+    httpMap.put("flushTimeOut", Long.parseLong(props.getProperty("com.metamx.emitter.http.flushTimeOut", "-1")));
```

Comment:
Can you add a comment here that it must be the same as `HttpEmitterConfig. DEFAULT_FLUSH_TIME_OUT`?